code/game/machinery/transformer.dm

```diff
@@ -100,10 +100,12 @@
 
 	H.emote("scream")
 	if(!masterAI) // If the factory was placed via admin spawning or other means, it wont have an owner_AI.
-		H.Robotize(robot_cell_type)
+		var/mob/living/silicon/robot/R = H.Robotize(robot_cell_type)
+		R.emagged = TRUE
```

Comment:
setting emagged isn't enough for borgs I'm pretty sure,

I don't believe their module will be properly emagged.

check the `emag_act()` proc for robots
```dm
				module.emag_act(user)
				module.module_type = "Malf" // For the cool factor
				update_module_icon()
				module.rebuild_modules() 
```

Reply:
Since they are modules-less and need to change their module, the new module will be emagged since they were emagged.

I've tested it with engi-borg and it had stun-prod